lib/ansible/config/base.yml

```diff
@@ -1474,18 +1473,20 @@ INTERPRETER_PYTHON_DISTRO_MAP:
 INTERPRETER_PYTHON_FALLBACK:
   name: Ordered list of Python interpreters to check for in discovery
   default:
-  - /usr/bin/python
   - python3.9
   - python3.8
   - python3.7
   - python3.6
   - python3.5
+  - /usr/bin/python3
   - python2.7
   - python2.6
+  - /usr/bin/python
   - /usr/libexec/platform-python
```

Comment:
This should probably also be above the py2 interpreters, since it should always be py3

Reply:
Roger. I wasn't sure if `platform-python` should _only_ be used as an absolute last resort.